lib/serializer/CTypeList.h

```diff
@@ -148,6 +148,8 @@ class DLL_LINKAGE CTypeList: public boost::noncopyable
 		return getTypeID(getTypeInfo(t), throws);
 	}
 
+	TypeInfoPtr getTypeDescriptor(ui16 typeID) const;
```

Comment:
What is this function for? Why is it in this PR?
(there is `TypeInfoPtr getTypeDescriptor(const std::type_info *type, bool throws = true) const;` already)

Reply:
Might be useful for debugging. Allows to get type name by its numeric id

Reply:
I have not found an existing equivalent

Reply:
I used it to get type name in requestRealized where we have only numeric id and it is hard to understand what request was actually realized. I can leave some logGlobal->trace there so that the function has at least some usage